packages/encoding/src/index.ts

```diff
@@ -1,12 +1,63 @@
 /**
  * Code modified from https://github.com/github/webauthn-json/blob/e932b3585fa70b0bd5b5a4012ba7dbad7b0a0d0f/src/webauthn-json/base64url.ts#L23
  */
+export const DEFAULT_JWK_MEMBER_BYTE_LENGTH = 32;
+
 export function stringToBase64urlString(input: string): string {
   // string to base64 -- we do not rely on the browser's btoa since it's not present in React Native environments
   const base64String = btoa(input);
   return base64StringToBase64UrlEncodedString(base64String);
 }
 
+export function base64urlToBuffer(baseurl64String: string): ArrayBuffer {
+  // Base64url to Base64
+  const padding = "==".slice(0, (4 - (baseurl64String.length % 4)) % 4);
+  const base64String =
+    baseurl64String.replace(/-/g, "+").replace(/_/g, "/") + padding;
+
+  // Base64 to binary string
+  const str = atob(base64String);
+
+  // Binary string to buffer
+  const buffer = new ArrayBuffer(str.length);
+  const byteView = new Uint8Array(buffer);
+  for (let i = 0; i < str.length; i++) {
+    byteView[i] = str.charCodeAt(i);
+  }
+  return byteView;
+}
+
+export function bufferToBase64url(buffer: ArrayBuffer): string {
```

Comment:
Same here. This doesn't seem to be used anywhere outside of the new tests.

Reply:
I was planning on consolidating various utils in `packages/encoding` instead of having the same util functions in `http` (specifically `packages/http/src/webauthn-json/base64url.ts`)

Reply:
ended up removing for now! maybe we can move it back, consolidate utilities in `encoding` down the line, along with a pureJS implementation of `atob` (similar to the `btoa` impl we have vendored in)